app/app.js

```diff
@@ -3,6 +3,12 @@ import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
+// adds "toggleCommentIndented" command to codemirror for default keymap
+
+if (CodeMirror) {
+  CodeMirror.keyMap.default['Cmd-/'] = 'toggleCommentIndented';
+}
```

Comment:
Can we move this to somewhere more appropriate, like an initalizer?

Reply:
👍